lib/configGenerator/generateTerraformConfig.js

```diff
@@ -5,6 +5,7 @@ async function generateTerraformConfig(network, networkName, masternodesCount) {
     node_count: 1,
     miner_count: 1,
     masternode_count: masternodesCount,
+    main_domain: '"networks.dash.org"',
```

Comment:
I guess we shouldn't set dash domain as default value since we expect other organizations and devs use it